themes/mlibrary_new/custom.php

```diff
@@ -20,6 +20,32 @@
 * Create the information in cards in introduction page, image with title and text
 *
 **/
+
+function get_most_used_tags_in_exhibits()
+{
+  $db = get_db();
+  $select = "
```

Comment:
Is there anything preventing use of the database abstraction provided by zend?

Reply:
This is the best I could find to merge three tables and get the query. I will create a new issue for it and post it to Omeka community.

Reply:
These queries aren't that involved, did you try:
```
$select = $db->select()
  ->from(...)
  ->join(...)
  ->join(...)
  ->group(...)
  ->order(...)
```